guides/source/active_record_basics.md

```diff
@@ -220,134 +263,192 @@ The application will have to use the [`id_value`][] alias attribute to access th
-users = User.where(name: 'David', occupation: 'Code Artist').order(created_at: :desc)
+# Find all books with a given an author, sort by created_at in reverse chronological order
+hitchhiker_series = Book.where(author: "Douglas Adams").order(created_at: :desc)
+```
+
+resulting in this SQL:
+
+```sql
+SELECT "books".* FROM "books" WHERE "books"."author" = ? ORDER BY "books"."created_at" DESC [["author", "Douglas Adams"]]
 ```
 
-You can learn more about querying an Active Record model in the [Active Record
-Query Interface](active_record_querying.html) guide.
+There are many more Active Record methods to read and query records. You can learn more about them in the [Active Record Query Interface](active_record_querying.html) guide.
 
 ### Update
 
 Once an Active Record object has been retrieved, its attributes can be modified
 and it can be saved to the database.
 
 ```ruby
-user = User.find_by(name: 'David')
-user.name = 'Dave'
-user.save
+book = Book.find_by(author: "J.R.R. Tolkien")
+book.title = "The Lord of the Rings: The Fellowship of the Ring"
+book.save
 ```
 
 A shorthand for this is to use a hash mapping attribute names to the desired
 value, like so:
 
 ```ruby
-user = User.find_by(name: 'David')
-user.update(name: 'Dave')
+book = Book.find_by(author: "J.R.R. Tolkien")
+book.update(title: "The Lord of the Rings: The Fellowship of the Ring")
 ```
 
-This is most useful when updating several attributes at once.
+the `update` results in the following SQL:
+
+```sql
+ UPDATE "books" SET "title" = ?, "updated_at" = ? WHERE "books"."id" = ?  [["title", "The Lord of the Rings: The Fellowship of the Ring"], ["updated_at", "2024-02-22 20:51:13.487064"], ["id", 104]]
+```
+
+This is useful when updating several attributes at once. Similar to `create`, using `update` will commit the updated records to the database.
 
 If you'd like to update several records in bulk **without callbacks or
 validations**, you can update the database directly using `update_all`:
 
 ```ruby
-User.update_all max_login_attempts: 3, must_change_password: true
+Book.update_all status: "already own"
 ```
 
 ### Delete
 
-Likewise, once retrieved, an Active Record object can be destroyed, which removes
-it from the database.
+Likewise, once retrieved, an Active Record object can be destroyed, which removes it from the database.
 
 ```ruby
-user = User.find_by(name: 'David')
-user.destroy
+book = Book.find_by(author: "J.R.R. Tolkien")
+book.destroy
+```
+
+The `destroy` results in this SQL:
+
+```sql
+DELETE FROM "books" WHERE "books"."id" = ?  [["id", 104]]
 ```
 
 If you'd like to delete several records in bulk, you may use `destroy_by`
 or `destroy_all` method:
 
 ```ruby
-# find and delete all users named David
-User.destroy_by(name: 'David')
+# Find and delete all books by Douglas Adams
+Book.destroy_by(author: "Douglas Adams")
 
-# delete all users
-User.destroy_all
+# Delete all books
+Book.destroy_all
 ```
 
 Validations
```

Comment:
I think the current examples under these following sections are fine, but would it make sense to use `Book` instead of `User` with them as well?

Reply:
That's odd, my diff already has `Book` for destroy section. Something might be outdated.

For the top level sections like Validations, I think a little variation can be okay. But within a section like `CRUD` keeping it all to `Book` makes a lot of sense.